Kernel/Process.cpp

```diff
@@ -743,42 +768,12 @@ void create_signal_trampolines()
     auto* trampoline_region = MM.allocate_user_accessible_kernel_region(PAGE_SIZE, "Signal trampolines").leak_ref();
     g_return_to_ring3_from_signal_trampoline = trampoline_region->vaddr();
 
+    u8* trampoline = (u8*)asm_signal_trampoline;
+    u8* trampoline_end = (u8*)asm_signal_trampoline_end;
+    size_t trampoline_size = trampoline_end - trampoline;
```

Comment:
This is a much nicer way of doing this. Now I feel silly for writing out all the opcodes manually :)